.gitlab-ci.yml

```diff
@@ -1,6 +1,29 @@
 stages:
+  - vulnerabilities
   - test
 
+######################################
+# Vulnerabilities stage related jobs #
+######################################
+check php vendors:
+  stage: vulnerabilities
+  image: sumocoders/framework-php72
```

Comment:
Het maak op zich niet echt een verschil in die geval, maar het zou geen slecht idee zijn om een `sumocoders/forkcms-php72` image aan te maken en hier gebruiken. Gewoon voor consistentie.